Elbow Saver/Shapes/TimerArc.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TimerArc.swift
+//  TimerArc
+//
+//  Created by Daniel Luo on 8/23/21.
+//
+
+import SwiftUI
+
+struct TimerArc: Shape {
+    var secondsRemaining: Double
+    let totalSeconds: Double
```

Comment:
These can be private yea?

Reply:
Actually I don't think so in this case because the parent view manipulates these values to draw the shape